src/main/java/com/googlecode/download/maven/plugin/internal/WGet.java

```diff
@@ -625,4 +635,26 @@ private boolean useHttpProxy(final ProxyInfo proxyInfo) {
         }
         return result;
     }
+
+    /**
+     * @return {@link FileMapper}s to be used for rewriting each target path, or {@code null} if no rewriting shall
+     *         happen.
+     *
+     * @since 1.6.8
+     */
+    public FileMapper[] getFileMappers()
```

Comment:
@mkarg do we really need getter/setter for `fileMappers`?

Reply:
I don't know actually if the solution still works without. I simply copied from the dependency plugin source code.

Reply:
If you want me to remove it, please clearly tell me.

Reply:
It should work, so, please, remove )

Reply:
Just tested and confirm that it passes test without, so I removed it in https://github.com/maven-download-plugin/maven-download-plugin/pull/206/commits/d7b02e0f5d5ce439a9e1c54e8c7c842803606434.